### PR TITLE
Add Professor Layton And The Azran Legacy

### PR DESCRIPTION
**Pull request by letuswar**
Adds Professor Layton And The Azran Legacy.

*Automatically generated by PR builder*

### CODE REVIEW

File: games/professor-layton-and-the-azran-legacy/game.dat [hunk 1]
```diff
@@ -0,0 +1,19 @@
+title = "Professor Layton And The Azran Legacy"
+description = "Professor Layton and the Azran Legacy is a puzzle game developed by Level-5 for the Nintendo 3DS. It is the sixth entry in the Professor Layton series, making up the third and final part of the prequel trilogy of games and according to Level 5 CEO Akihiro Hino, it will be the last Layton title to star Layton himself as the protagonist. The game saw a release in early to late 2013 in all territories except North America, where it was released on February 28 of the following year."
```
Comment (Leo626) [inline games/professor-layton-and-the-azran-legacy/game.dat]: The last sentence with the the release dates is redundant.

File: games/professor-layton-and-the-azran-legacy/game.dat [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+needs_system_files = true
+needs_shared_font = true
+
+[[releases]]
+title = "0004000e000f3000"
```
Comment (Leo626) [inline games/professor-layton-and-the-azran-legacy/game.dat]: For consistency, the title ID should have capital letters.

File: games/professor-layton-and-the-azran-legacy/game.dat [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+region = "eur"
+release_date = "2013-11-08"
+
+[[testcases]]
+title = "0004000e000f3000"
```
Comment (Leo626) [inline games/professor-layton-and-the-azran-legacy/game.dat]: Ditto.

File: games/professor-layton-and-the-azran-legacy/game.dat [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+compatibility = "0"
+date = "2017-11-20"
+version = "HEAD-de0af83"
+cpu = "AMD Ryzen 5 1600"
+gpu = "GeForce GTX 1060 6GB"
```
Comment (Leo626) [inline games/professor-layton-and-the-azran-legacy/game.dat]: `Nvidia GeForce GTX 1060 6GB`

File: games/professor-layton-and-the-azran-legacy/game.dat (hunk 2 above)
Comment (Hexagon12) [inline games/professor-layton-and-the-azran-legacy/game.dat]: That's not the correct title ID. The correct one is "00040000000F3000".

File: games/professor-layton-and-the-azran-legacy/game.dat (hunk 3 above)
Comment (Hexagon12) [inline games/professor-layton-and-the-azran-legacy/game.dat]: Ditto.

**Comment by TheKoopaKingdom** [comment]
Closed for inactivity.

**Pull request closed without merging**